enigma-cli/src/test/java/org/quiltmc/enigma/command/FillClassMappingsCommandTest.java

```diff
@@ -50,7 +50,7 @@ public void test() throws Exception {
 		Path resultFile = Files.createTempFile("fillClassMappings", ".mappings");
 		FillClassMappingsCommand.run(JAR, MAPPINGS, resultFile, false, null, null);
 
-		MappingsReader reader = CommandsUtil.getReader(createEnigma(), resultFile);
+		MappingsReader reader = CommandsUtil.getReader(Enigma.create(), resultFile);
```

Comment:
Wrong merge?

Reply:
this is a test and doesn't have access to that method

Reply:
oh whoops